test_files/export/export.tsickle.ts

```diff
@@ -16,4 +19,4 @@ let /** @type {number} */ export2 = 3;
 
 // This is just an import, so export5 should still be included.
 import {export5} from './export_helper';
-const export5: NeverTypeCheckMe = export5;  /* local alias for Closure JSDoc */
+const tsickle_module_prefix_4 = goog.require('test_files.export.export_helper');
```

Comment:
So we end up importing every module twice?  I agree this is allowed by the Closure semantics but it's a bit disappointing.